docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -180,7 +180,7 @@ Synopsis:
 
 ```
 LISTAGG( expression [, separator] [ON OVERFLOW overflow_behaviour])
-    WITHIN GROUP (ORDER BY sort_item, [...])
+    WITHIN GROUP (ORDER BY sort_item, [...]) [FILTER (WHERE condition)]
```

Comment:
It is important to document the synopsis of the function to help out potential users with a concise description of its syntax.

Reply:
Great catch!